NewEssentials/Chat/BroadcastingService.cs

```diff
@@ -0,0 +1,73 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Cysharp.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using OpenMod.API.Ioc;
+using OpenMod.API.Plugins;
+using OpenMod.Core.Helpers;
+using SDG.Unturned;
+using UnityEngine;
+
+namespace NewEssentials.Chat
+{
+    [ServiceImplementation]
+    public class BroadcastingService : IBroadcastingService
+    {
+        public bool IsActive { get; set; }
+
+        private IConfiguration m_Configuration;
+        private int m_BroadcastIndex;
+        private IOpenModPlugin _plugin;
+
+        public BroadcastingService(IConfiguration config, IOpenModPlugin plugin)
```

Comment:
Can not use IOpenModPlugin here, must use IPluginAccessor<NewEssentials> instead